cmd/dep/vndr_importer.go

```diff
@@ -123,7 +124,10 @@ func (v *vndrImporter) convert(pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, er
 		fb.NewLockedProjectFeedback(lp, fb.DepTypeImported).LogFeedback(v.logger)
 	}
 
-	return manifest, lock, nil
+	if len(manifest.Constraints) > 0 {
```

Comment:
You never need to apologize! 😀 We all have families, jobs, hobbies and *gasp* other interests.